createdisk.sh

```diff
@@ -83,6 +83,9 @@ ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} 'sudo bash -x -s' <<EOF
   systemctl enable gvisor-tap-vsock.service
 EOF
 
+# Preload routes controller
+${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo crictl pull quay.io/crcont/routes-controller:6effc2f0304d3a11c1e22c278b10630e834d0220'
```

Comment:
no version?

Reply:
We are using sha so do we still need version?

Reply:
release created, v0.0.1